www/c-icap/src/opnsense/mvc/app/models/OPNsense/CICAP/General.xml

```diff
@@ -0,0 +1,59 @@
+<model>
+    <mount>//OPNsense/cicap/general</mount>
+    <description>c-icap configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <timeout type="IntegerField">
+            <default>300</default>
+            <Required>Y</Required>
+        </timeout>
+        <maxkeepaliverequests type="IntegerField">
+            <default>100</default>
+            <Required>Y</Required>
+        </maxkeepaliverequests>
+        <keepalivetimeout type="IntegerField">
+            <default>600</default>
+            <Required>Y</Required>
+        </keepalivetimeout>
+        <startservers type="IntegerField">
+            <default>3</default>
+            <Required>Y</Required>
+        </startservers>
+        <maxservers type="IntegerField">
+            <default>10</default>
+            <Required>Y</Required>
+        </maxservers>
+        <minsparethreads type="IntegerField">
+            <default>10</default>
+            <Required>Y</Required>
+        </minsparethreads>
+        <maxsparethreads type="IntegerField">
+            <default>20</default>
+	    <Required>Y</Required>
+	</maxsparethreads>
+        <threadsperchild type="IntegerField">
+            <default>10</default>
+	    <Required>Y</Required>
+	</threadsperchild>
+        <maxrequestsperchild type="IntegerField">
+            <default>0</default>
+	    <Required>Y</Required>
+	</maxrequestsperchild>
+        <listenaddress type="TextField">
+            <default>127.0.0.1</default>
+            <Required>Y</Required>
+        </listenaddress>
+        <serveradmin type="TextField">
+            <default>you@your.address</default>
+            <Required>Y</Required>
+        </serveradmin>
+        <servername type="TextField">
+            <default>YourServerName</default>
+            <Required>Y</Required>
+        </servername>
```

Comment:
This was briefly discussed in collects plugin... not strictly needed, we should try the system default if not set, yes. For some use cases it may make sense to separate hostnames.